### PR TITLE
officially support redis using requirements

### PR DESCRIPTION
**Pull request by atarkowska**
This PR adds official support for redis, see https://trello.com/c/GIJQA0kg/177-redis-out-of-experimental

### CODE REVIEW

File: omero/sysadmins/unix/install-web.txt [hunk 1]
```diff
@@ -122,14 +122,14 @@ OMERO.web maintenance
...
 
-      - `Memcached <https://memcached.org/>`_::
+      - `Redis 2.8+ <http://redis.io/>`_ requires `django-redis 4.4+ <http://niwinz.github.io/django-redis/latest/>`_::
 
-          $ bin/omero config set omero.web.caches '{"default": { "BACKEND": "django.core.cache.backends.memcached.MemcachedCache", "LOCATION": "127.0.0.1:11211", "TIMEOUT": "86400" } }'
+          $ pip install OMERO.py/share/web/requirements-redis.txt
+          $ bin/omero config set omero.web.caches '{"default": {"BACKEND": "django_redis.cache.RedisCache","LOCATION": "redis://redis:6379/0"}}'
 
-      - EXPERIMENTAL: `Redis 2.4+ <http://redis.io/>`_ requires `django-redis-cache 1.6.5+ <http://django-redis-cache.readthedocs.org/en/latest/intro_quick_start.html>`_::
+      - DEPRECATED: `Memcached <https://memcached.org/>`_::
```
Comment (atarkowska) [inline omero/sysadmins/unix/install-web.txt:130]: Deprecate or remove completely?

Comment (joshmoore) [reply omero/sysadmins/unix/install-web.txt:130]: Perhaps something that there are "alternatives". Are there others supported by django that someone could be encouraged to investigate?

Comment (atarkowska) [reply omero/sysadmins/unix/install-web.txt:130]: in addition to what we have in settings by default, I can only think of memcached. https://docs.djangoproject.com/en/1.8/topics/cache/

**Comment by hflynn** [comment]
Code PR merged, listing for review tomorrow

**Review by chris-allan — COMMENTED** [review]
Looks good to me. Up to you if you want to adjust the style of the example JSON target of `omero config set`. The Redis version is missing whitespace before the `LOCATION` key and the memcached version has differing leading and trailing whitespace surrounding the inner dictionary.

Some of the other caches are useful, such as local memory and database, (we're using the database one for a few things) but probably implementation specific and not something we actively want to support with out install documentation.

/cc @joshmoore

**Comment by joshmoore** [comment]
Makes sense. Good to know that we can add memory & database if need be. (Whitespace fixed) Leaving @hflynn the fun of wrangling travis.

**Pull request merged**